_includes/pbc/all/install-features/202204.0/marketplace/install-the-marketplace-merchant-portal-core-feature.md

```diff
@@ -0,0 +1,1255 @@
+    {
+        return [
+            new MerchantPortalAclEntityMetadataConfigExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Merchant/MerchantDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Merchant;
+
+use Spryker\Zed\AclMerchantPortal\Communication\Plugin\Merchant\MerchantAclMerchantPostCreatePlugin;
+use Spryker\Zed\Merchant\MerchantDependencyProvider as SprykerMerchantDependencyProvider;
+
+class MerchantDependencyProvider extends SprykerMerchantDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\MerchantExtension\Dependency\Plugin\MerchantPostCreatePluginInterface>
+     */
+    protected function getMerchantPostCreatePlugins(): array
+    {
+        return [
+            new MerchantAclMerchantPostCreatePlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/MerchantUser/MerchantUserDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\MerchantUser;
+
+use Spryker\Zed\AclMerchantPortal\Communication\Plugin\MerchantUser\AclMerchantPortalMerchantUserRoleFilterPreConditionPlugin;
+use Spryker\Zed\AclMerchantPortal\Communication\Plugin\MerchantUser\MerchantAclMerchantUserPostCreatePlugin;
+use Spryker\Zed\MerchantUser\MerchantUserDependencyProvider as SprykerMerchantUserDependencyProvider;
+
+class MerchantUserDependencyProvider extends SprykerMerchantUserDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\MerchantUserExtension\Dependency\Plugin\MerchantUserPostCreatePluginInterface>
+     */
+    protected function getMerchantUserPostCreatePlugins(): array
+    {
+        return [
+            new MerchantAclMerchantUserPostCreatePlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\MerchantUserExtension\Dependency\Plugin\MerchantUserRoleFilterPreConditionPluginInterface>
+     */
+    protected function getMerchantUserRoleFilterPreConditionPlugins(): array
+    {
+        return [
+            new AclMerchantPortalMerchantUserRoleFilterPreConditionPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that non-merchant users whose Acl Group has Back Office allowed Acl Group Reference (see `AclMerchantPortalConfig::getBackofficeAllowedAclGroupReferences()`) can log in to the Back Office.
+
+{% endinfo_block %}
+
+**src/Pyz/Zed/Acl/AclDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Acl;
+
+use Spryker\Zed\Acl\AclDependencyProvider as SprykerAclDependencyProvider;
+use Spryker\Zed\AclMerchantPortal\Communication\Plugin\MerchantUser\ProductViewerForOfferCreationAclInstallerPlugin;
+
+class AclDependencyProvider extends SprykerAclDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\AclExtension\Dependency\Plugin\AclInstallerPluginInterface>
+     */
+    protected function getAclInstallerPlugins(): array
+    {
+        return [
+            new ProductViewerForOfferCreationAclInstallerPlugin(),
+        ];
+    }
+}
+```
+
+#### Enable Merchant Portal infrastructural plugins
+
```

Comment:
Don't we need a table like [here](https://github.com/spryker/spryker-docs/pull/1857/files#diff-e973982423e49dc1ac14c61fe24ebb75eefcb0a27ce2ef857d6c14389e6e441bR92) for the plugins?